### DISCUSSION TITLE
Please, add GPU memory temperature sensors

### DISCUSSION THREAD

**Discussion by miguel1900** [Rem0o/FanControl.Releases#1136] [Ideas]
Hi! I'm still using this amazing tool after more than a year! :)

But I'm missing just one thing: GPU's Hot Spot temperature and Memory Temperature.

As RTX are quite memory-hot graphic cards, I think this would be a Must for GPU enthusiasts.

Thank you and best regards!
Miguel

**Comment by Rem0o** [comment]
Swap GPU provider. The LHM one does include it I believe. (Sensor settings)

**Reply by miguel1900** [reply]
Oh, thanks, you're right!!

I went to Sensor settings and unchecked the Enabled checkbox under NvApiWrapper.

**Comment by mxr-mxr-mxr** [comment]
Hey!  Im thinking GPU Memory temp too to fancontrolling. 
I have orginal Asus Rog Strix RTX3080 OC. (without LHR)

I can see GPU, GPU Hotspot and GPU Memory temperatures from Aida64 -app. 
Does these temperatures possible to integrate to FanControl app too? Fan Control app support now only GPU-temp with my Videocard.

**Reply by miguel1900** [reply]
Yeah, you can see those values following the described steps!
